openupgrade_scripts/scripts/sale_project/14.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,36 @@
+---Models in module 'sale_project'---
+---Fields in module 'sale_project'---
+sale_project / product.product          / project_id (many2one)         : previously in module sale_timesheet
+sale_project / product.product          / project_template_id (many2one): previously in module sale_timesheet
+sale_project / product.product          / service_tracking (selection)  : previously in module sale_timesheet
+sale_project / product.template         / project_id (many2one)         : previously in module sale_timesheet
+sale_project / product.template         / project_template_id (many2one): previously in module sale_timesheet
+sale_project / product.template         / service_tracking (selection)  : previously in module sale_timesheet
+sale_project / project.project          / sale_line_id (many2one)       : previously in module sale_timesheet
+sale_project / project.project          / sale_order_id (many2one)      : previously in module sale_timesheet
+sale_project / project.task             / sale_line_id (many2one)       : previously in module sale_timesheet
+sale_project / project.task             / sale_order_id (many2one)      : previously in module sale_timesheet
+sale_project / sale.order               / project_id (many2one)         : previously in module sale_timesheet
+sale_project / sale.order               / project_ids (many2many)       : previously in module sale_timesheet
+sale_project / sale.order               / tasks_count (integer)         : previously in module sale_timesheet
+sale_project / sale.order               / tasks_ids (many2many)         : previously in module sale_timesheet
+sale_project / sale.order               / visible_project (boolean)     : previously in module sale_timesheet
+sale_project / sale.order.line          / is_service (boolean)          : previously in module sale_timesheet
+sale_project / sale.order.line          / project_id (many2one)         : previously in module sale_timesheet
+sale_project / sale.order.line          / task_id (many2one)            : previously in module sale_timesheet
+# NOTHING TO DO
+
+---XML records in module 'sale_project'---
+NEW ir.actions.server: sale_project.project_task_server_action_batch_invoice
+NEW ir.model.access: sale_project.access_sale_order_line_project_manager [renamed from sale_timesheet module]
+NEW ir.model.access: sale_project.access_sale_order_project_manager [renamed from sale_timesheet module]
+NEW ir.model.constraint: sale_project.constraint_project_project_sale_order_required_if_sale_line [renamed from sale_timesheet module]
+NEW ir.rule: sale_project.sale_order_line_rule_project_manager [renamed from sale_timesheet module] (noupdate)
```

Comment:
I would call rename_xmlids() in pre.

Reply:
I doubt it. Maybe you are seeing the new xmlid, but the old still exists.

It's better to assure the module rename in pre-migration. you may use openupgrade.update_module_names() instead of openupgrade.rename_xmlids(), as you prefer.

Reply:
I verified that there is just one record in the database, that it is not duplicated, and renamed. But for sure, I'll include a code renaming for them.